resources/vue/src/components/account/common/NavLink.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+    <li>
+        <router-link :to="link" class="nav-link link-dark">
+            {{ title }}
+        </router-link>
+    </li>
```

Comment:
This component seems rather useless, you can inline this one in the component that use it. It's also kind of tied to be inside `ul` to maintain the validity of HTML, so it's not like it's all that good for re-use anyway. This component would be worth splitting if it wasn't tied to `ul` and also had some specific styling or data.